h2o-algos/src/main/java/hex/tree/isofor/IsolationForestModel.java

```diff
@@ -168,6 +169,10 @@ static double normalizePathLength(double pathLength, int minPathLength, int maxP
     throw new UnsupportedOperationException("Isolation Forest support only MOJOs.");
   }
 
+  @Override protected CategoricalEncoding getGenModelEncoding() {
```

Comment:
unnecessary

Reply:
hm but then I need to make it public in SharedTreeModel.java, otherwise I'll have error when using it in hex/tree/isofor/IsolationForestMojoWriter.java

Reply:
oh I see, this is to give visibility to IsolationForestMojoWriter

Reply:
yes so i don't know whether it's better to make it public in SharedTreeModel or override like this in IsolationForestModel